app/upgrades.go

```diff
@@ -34,40 +27,18 @@ type appUpgrade struct {
 }
 
 var handlers = map[string]appUpgrade{
-	"lava":  {}, // upgrade for 1.10.0
-	"mango": {}, // upgrade for 1.11.1
+	"lava": {}, // upgrade for 1.10.0
+	"mango": {
+		Handler: func(app *App, ctx sdk.Context, plan upgradetypes.Plan) (module.VersionMap, error) {
+			params := app.MsgFeesKeeper.GetParams(ctx)
+			app.MsgFeesKeeper.SetParams(ctx, params)
```

Comment:
Do we need to actually set the value here? This appears to just be doing a read/write without any changes.

Reply:
Yes, we do.  SetParams sets to the paramstore, while GetParams from the msgfeeskeeper is getting the params from the param store or returning the default value.